docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -25,35 +25,40 @@ The following table shows how risk levels are applied to a host, based on the no
 |==============================================
 
 [discrete]
-=== Deploy the host risk score package
+=== Deploy host risk score
 
-To deploy the host risk score framework in your environment, follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md[these steps]. To view host risk score data in the {security-app}, you must https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md#8-enable-kibana-features[enable the `riskyHostsEnabled` feature flag]. However, enabling the feature flag is *not* required to view the Lens dashboards.
-
-NOTE: Update host risk score artifacts after you upgrade the {stack}. To do this, download a release bundle that's compatible with the new {stack} version and repeat all the steps referenced above. Failure to do so might cause views in the {security-app} to break.
-
-[[view-host-risk-score]]
-[discrete]
-=== View host risk score data
-
-If the `riskyHostsEnabled` feature flag is enabled:
+To deploy the host risk score framework in your environment:
 
+. https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/host-risk-score.md#8-enable-kibana-features[Enable the `riskyHostsEnabled` feature flag].
 . In the {security-app}, go to *Dashboards* -> *Overview*, then locate the *Current host risk scores* card in the lower-right corner.
-. Click *View dashboard*.
 +
 [role="screenshot"]
-image::images/host-score-overview.png[]
+image::images/host-risk-score-enable-dev-tools.png[]
+. Click the *Enable via Dev Tools* button, which will take you to the Dev Tools Console in {kib}, pre-populated with all the scripts required to enable host risk score.
```

Comment:
A twist on @nastasha-solomon's edits, using streamlined present tense instead of future:
```suggestion
. Click *Enable via Dev Tools*. This opens the Dev Tools Console in {kib} and populates it with the scripts required to enable the host risk score feature.
```